gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -230,6 +230,9 @@ const CssVariablesPage = () => {
               as the value of a <NxCode>filter</NxCode> <NxCode>drop-shadow</NxCode>.
             </NxWarningAlert>
           </PropertyDocItem>
+          <PropertyDocItem propertyVar="--nx-radio-checkbox-border-width">
+            The standard checkbox border width, or radio stroke-width.
```

Comment:
No longer using `stroke-width`

Reply:
> No longer using `stroke-width`

Fixed as part of: fa351f5ea157e03267d26df3eebf91cfb2881237